soh/soh/GameMenuBar.cpp

```diff
@@ -163,8 +163,8 @@ namespace GameMenuBar {
                 ImGui::PopStyleColor(1);
                 ImGui::PopStyleVar(3);
             #ifndef __SWITCH__
-                UIWidgets::PaddedEnhancementCheckbox("Use Controller Navigation", "gControlNav", true, false);
-                UIWidgets::Tooltip("Allows controller navigation of the menu bar\nD-pad to move between items, A to select, and X to grab focus on the menu bar");
+                UIWidgets::PaddedEnhancementCheckbox("Menubar Controller Navigation", "gControlNav", true, false);
+                UIWidgets::Tooltip("Allows controller navigation of the SOH menu bar (Settings, Enhancements,...)\nCAUTION: This will disable your game input until the menu bar is hidden\n\nD-pad to move between items, A to select, and X to grab focus on the menu bar");
```

Comment:
Nitpicking, but I think I'd prefer `CAUTION: This will disable game inputs while the menubar is visible.`